app/models/hbx_enrollment.rb

```diff
@@ -2724,7 +2724,11 @@ def primary_enrollee?
   end
 
   def couple_enrollee?
-    @couple_enrollee ||= primary_enrollee? && hbx_enrollment_members.any? {|hem| hem.primary_relationship == 'spouse'}
+    if EnrollRegistry.feature_enabled?(:domestic_partner_rating)
+      @couple_enrollee ||= primary_enrollee? && hbx_enrollment_members.any? {|hem| (hem.primary_relationship == 'spouse') || (hem.primary_relationship == 'domestic_partner') }
+    else
+      @couple_enrollee ||= primary_enrollee? && hbx_enrollment_members.any? {|hem| (hem.primary_relationship == 'spouse')}
```

Comment:
these lines seem to be almost the same -- how could we eliminate some of the duplicate code?